site/src/sphinx/client-retry.rst

```diff
@@ -211,7 +214,27 @@ it is timed out by the per-attempt timeout.
                                            10,000ms (ResponseTimeoutException)
     @endditaa
 
+In above example, every attempt is made before it is timed out because the :api:`Backoff` is disabled.
+However, What if a :api:`Backoff` is enabled and the moment of trying next attempt is after the point of
+occurring :api:`ResponseTimeoutException`? The :api:`RetryingClient` do not schedule for the next attempt and
+returned with the latest :api:`Response` from the server immediately. Consider following example:
+
+.. uml::
+
+    @startditaa(--no-separation, --no-shadows, scale=0.95)
+    0ms         3,000ms     6,000ms     9,000ms     12,000ms
+    |           |           |           |           |
+    +-----------+-----------+-----------+-----------+--------------------+
+    | Attempt 1 |           | Attempt 2 |           | Attempt 3 is made? |
+    +-----------+-----------+-----------+----+------+--------------------+
+                                             |
+                                           10,000ms (ResponseTimeoutException)
+    @endditaa
 
+Unlike above example, the :api:`Backoff` is enabled and it makes the :api:`RetryingClient` retries after ``3``
+seconds. When the second attempt is finished, the next attempt will be 12,000ms which is after 10,000ms.
+The :api:`RetryingClient`, at this point, stops to retry and returned with the latest :api:`Response` which it
```

Comment:
- stops to retry -> stops retrying
- returned with the latest Response -> finishes the retry session with the last Response